src/openxr/OpenXRApi.h

```diff
@@ -52,6 +52,12 @@
 // forward declare this
 class OpenXRApi;
 
+enum TrackingConfidence {
+	TRACKING_CONFIDENCE_NO,
```

Comment:
This should be `TRACKING_CONFIDENCE_NONE` to match the version in the Godot source code.

Reply:
Good point, done!